<launch>
    <!-- 启动乌龟节点与键盘控制节点 -->
    <node pkg = "turtlesim" type = "turtlesim_node" name = "turtle1" output = "screen" />
    <node pkg = "turtlesim" type = "turtle_teleop_key" name = "key_control" output = "screen" />
    <!-- 启动创建第二只乌龟的节点 -->
    <node pkg = "demo_tf" type = "client_turtle_set" name = "turtle2" output = "screen" />
    <!-- 启动两个坐标发布节点 -->
    <node pkg = "demo_tf" type = "turtle_frames_pub" name = "t1" output = "screen" args = "turtle1" />
    <node pkg = "demo_tf" type = "turtle_frames_pub" name = "t2" output = "screen" args = "turtle2" />
    <!-- 启动坐标转换节点 -->
    <node pkg = "demo_tf" type = "turtle_frames_sub" name = "listener" output = "screen" />
</launch>